api-configuration/src/main/java/org/jvalue/ods/apiConfigurationService/endpoint/ApiConfigurationService.java

```
@@ -0,0 +1,143 @@
+package org.jvalue.ods.apiConfigurationService.endpoint;
+
+import org.jvalue.ods.apiConfigurationService.helper.GraphqlQueryFactory;
+import org.jvalue.ods.apiConfigurationService.helper.SqlFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.Arrays;
+
+import static org.jvalue.ods.apiConfigurationService.helper.GraphqlQueryFactory.createSQLGraphQuery;
+
+@RestController
+@RequestMapping("/")
+public class ApiConfigurationService implements ApiConfigurationServiceI {
+
+  private static final String HASURA_ADMIN_ENDPOINT = "http://172.17.0.1:7654/v1/query";
+  private static final String DEFAULT_SCHEMA = "storage";
```

Also via env variable. By the way, make sure to not with default values in order to fail fast and make visible that the service is not configured right.